### ISSUE TITLE
Multiple hyphens in attribute

### ISSUE THREAD

**Issue by mahemoff** [k0kubun/hamlit#80]
Hi, I found an issue I couldn't see on Limitations.

While regular hyphenated attribs seem to work fine (e.g. `%div(data-product='book')`), I found attribs with multiple hyphens seem to fail.

For example, this input:

    %div(data-product-type='book')

leads to this output:

   `<div data product type='book'></div>`

Expected: 

   `<div data-product-type='book'></div>`

**Comment by WaKeMaTTa** [comment]
```ruby
%div{"data-product-type" => "book"}
```

**Comment by mahemoff** [comment]
Thanks, I did port to a hash in the end, and found it necessary in a few cases.

I guess () style just isn't advised with Hamlit in general?

**Comment by k0kubun** [comment]
Which version do you use?

**Comment by k0kubun** [comment]
```bash
~/src/.../k0kubun/hamlit $ hamlit       
rbenv: hamlit: command not found
~/src/.../k0kubun/hamlit $ cat in.haml                                                         
%div(data-product-type='book')

~/src/.../k0kubun/hamlit $ gem install hamlit -v 1.6.5                                                      
Fetching: hamlit-1.6.5.gem (100%)
Successfully installed hamlit-1.6.5
Parsing documentation for hamlit-1.6.5
Installing ri documentation for hamlit-1.6.5
Done installing documentation for hamlit after 0 seconds
1 gem installed
~/src/.../k0kubun/hamlit $ hamlit in.haml                                                                             
<div data product type='book'></div>

~/src/.../k0kubun/hamlit $ gem install hamlit -v 1.6.6                                                         
Fetching: hamlit-1.6.6.gem (100%)
Successfully installed hamlit-1.6.6
Parsing documentation for hamlit-1.6.6
Installing ri documentation for hamlit-1.6.6
Done installing documentation for hamlit after 0 seconds
1 gem installed
~/src/.../k0kubun/hamlit $ hamlit in.haml                                                                                
<div data-product-type='book'></div>

~/src/.../k0kubun/hamlit $ gem install hamlit                                                                    
Fetching: hamlit-2.5.0.gem (100%)
Building native extensions.  This could take a while...
Successfully installed hamlit-2.5.0
Parsing documentation for hamlit-2.5.0
Installing ri documentation for hamlit-2.5.0
Done installing documentation for hamlit after 0 seconds
1 gem installed
~/src/.../k0kubun/hamlit $ hamlit in.haml                                                                             
<div data-product-type='book'></div>
```

It may be fixed since v1.6.6.

**Comment by mahemoff** [comment]
I see, thanks. I did have a pre-1.66 version (v1.5.4). Verified this works after updating to 2.5.0.
